templates/kubelet_args.j2

```diff
@@ -0,0 +1,4 @@
+{{ kubelet_orig_file_content }}
```

Comment:
This would not work, please, test it out locally before submitting:

```suggestion
{{ kubelet_orig_file_content.content | | b64decode }}
```

Reply:
https://docs.ansible.com/ansible/latest/collections/ansible/builtin/slurp_module.html#return-values